src/base/solvers/solveCobraLP.m

```diff
@@ -1228,13 +1228,59 @@
             ILOGcplex.solve();
 
             origStat   = ILOGcplex.Solution.status;
+            stat = origStat;
             if origStat==1
                 f = osense*ILOGcplex.Solution.objval;
                 x = ILOGcplex.Solution.x;
                 w = ILOGcplex.Solution.reducedcost;
-                y = ILOGcplex.Solution.dual;
+                y = ILOGcplex.Solution.dual;                                            
+            elseif origStat == 4
+                %This is likely unbounded, but could be infeasible 
+                %Lets check, by solving an additional LP with a bounded
+                %objective.
+                %Store the original solution
+                Solution = ILOGcplex.Solution;
```

Comment:
I suggest we print out a message to the user that the status is `4` and that the problem is likely unbounded rather than solving another optimisation problem. We can suggest how to check for unboundedness or for infeasibility.